vulcan/lib/server.rb

```diff
@@ -16,7 +16,7 @@ class Server < Etna::Server
     end
 
     get 'api/:project_name/workflows', action: 'workflows#fetch', as: :workflows_view, auth: { user: { can_view?: :project_name } }
-    get 'api/:project_name/workflows/:workflow_name/:data', action: 'data#fetch', as: :data_view, auth: { user: { can_view?: :project_name } }
+    get 'api/:project_name/data/:cell_hash/:data_filename', action: 'data#fetch', as: :data_view, auth: { user: { can_view?: :project_name } }
     post 'api/:project_name/workflows/:workflow_name/:status', action: 'workflows#submit', as: :submit_view, auth: { user: { can_view?: :project_name } }
```

Comment:
For a real API, I think this should actually just become `/api/:project_name/workflows/:workflow_name`...I was trying to get different status JSONs back, but not sure how to mock that out...

Reply:
Yeah, that makes sense.  Part of my goal of having an e2e integration is so that I can pass this back to you and we can start actually shaping the API the way you'd like, with a (somewhat) backing skeleton in place.  Let me make some more progress and then we can work together and shape the api further.